src/components/EmptyResults.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  
+  <div class="col text-center no-found">
+    <h2 class="text-light font-weight-light">
+      No films found
+    </h2>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  </div>
+
+</template>
+
+<script>
+
+export default {
+  name: 'EmptyResults'
+};
+
+</script>
+
+<style scoped>
```

Comment:
in shared components not always useful use scoped styles
https://www.youtube.com/watch?v=x78G5U2C3zk epamer guy